tools/run_tests/run_tests.py

```diff
@@ -759,9 +759,13 @@ def _start_port_server(port_server_port):
     env = dict(os.environ)
     env['BUILD_ID'] = 'pleaseDontKillMeJenkins'
     if platform.system() == 'Windows':
+      # Working directory of port server needs to be outside of Jenkins
+      # workspace to prevent file lock issues.
```

Comment:
K... Let's bump the port_server version with this PR so the change gets picked up